robottelo/hosts.py

```diff
@@ -798,6 +798,17 @@ def register(
         cmd = target.satellite.cli.HostRegistration.generate_command(options)
         return self.execute(cmd.strip('\n'))
 
+    def api_register(self, target, **kwargs):
+        """Register a content host using the API.
```

Comment:
I feel like we could either name this something related to global registration like `def globally_register` or leave the name as is and update the description to mention global registration like `Register a content host using Global Registration.`

Reply:
Yeah it's not a great name that's why I thought at least the description :smile:

Reply:
+1 to keep the name and extend docstring

Reply:
+1 to this. Tying this in explicitly to global registration should make it more clear why this is the method being used going forward.